rgw/v2/tests/s3_swift/reusable.py

```diff
@@ -1590,42 +1586,73 @@ def prepare_for_bucket_lc_transition(config):
             zone = "secondary"
     else:
         zone = zonegroup = "default"
-    if config.ec_pool_transition:
-        utils.exec_shell_cmd(
-            f"radosgw-admin zonegroup placement add  --rgw-zonegroup {zonegroup} --placement-id default-placement --storage-class {ec_storage_class}"
-        )
-        utils.exec_shell_cmd(
-            f"radosgw-admin zone placement add --rgw-zone {zone} --placement-id default-placement --storage-class {ec_storage_class} --data-pool {ec_pool_name}"
-        )
-        utils.exec_shell_cmd(
-            "ceph osd erasure-code-profile set rgwec01 k=4 m=2 crush-failure-domain=osd crush-device-class=hdd"
-        )
-        utils.exec_shell_cmd(
-            f"ceph osd pool create {ec_pool_name} 32 32 erasure rgwec01"
-        )
-        utils.exec_shell_cmd(f"ceph osd pool application enable {ec_pool_name} rgw")
-    else:
-        utils.exec_shell_cmd(
-            f"radosgw-admin zonegroup placement add  --rgw-zonegroup {zonegroup} --placement-id default-placement --storage-class {storage_class}"
-        )
-        utils.exec_shell_cmd(
-            f"radosgw-admin zone placement add --rgw-zone {zone} --placement-id default-placement --storage-class {storage_class} --data-pool {pool_name}"
-        )
-        utils.exec_shell_cmd(f"ceph osd pool create {pool_name}")
-        utils.exec_shell_cmd(f"ceph osd pool application enable {pool_name} rgw")
-        if config.multiple_transitions:
-            second_pool_name = config.second_pool_name
-            second_storage_class = config.second_storage_class
-            utils.exec_shell_cmd(f"ceph osd pool create {second_pool_name}")
+    if config.test_ops.get("test_pool_transition", False):
```

Comment:
@viduship 
this is a new config variable right, it might fail for all existing configs as we gave the default value as False, could you make the default value as True and provide one sample pass log

if config.test_ops.get("test_pool_transition", True):

Reply:
right, thanks for adding that @hmaheswa . I would update and provide the regression tests for lc transition

Reply:
done